src/js/comments.js

```diff
@@ -0,0 +1,22 @@
+import * as API from './api.js';
+
+const displayCommentsPopup = async ({ mealId }) => {
+  const meal = await API.getMealById(mealId);
+  const modal = document.querySelector('.modal');
```

Comment:
The meal details should go inside the `.modal-content` container instead of `.modal`. 👍